### PR TITLE
Feat: MapleWebServer & UGC Profile handler

### LINKED ISSUE #282: Implement portraits
**Issue:**
Portraits/Icons/Logos/Avatars, whatever you want to call it, aren't saving. Most likely because it's to do with a UGC server so I guess this would fall under a feature more than a bug? Put it here anyway though :)

**Video Example:**
https://youtu.be/ADi2dEZOd-I

### PR DESCRIPTION
**Pull request by AngeloTadeucci**
- WebApi for UGC profiles;
- WIP UgcHandler implementation;
- Moved Dotenv.cs and Paths.cs to Maple2Storage;
- Resolves #282

### CODE REVIEW

File: MapleServer2/PacketHandlers/Login/LoginHandler.cs [hunk 1]
```diff
@@ -73,7 +73,7 @@ public override void Handle(LoginSession session, PacketReader packet)
 
                     Logger.Debug("Initializing login with account id: {session.AccountId}", session.AccountId);
                     session.Send(LoginResultPacket.InitLogin(session.AccountId));
-                    session.Send(UgcPacket.SetEndpoint("http://127.0.0.1/ws.asmx?wsdl", "http://127.0.0.1"));
+                    session.Send(UgcPacket.SetEndpoint("http://localhost:3000/ws.asmx?wsdl", "http://localhost:3000"));
```
Comment (AlanMorel) [inline MapleServer2/PacketHandlers/Login/LoginHandler.cs:76]: Can these URLs be thrown into the .env file?

File: MapleServer2/Servers/Login/LoginSession.cs [hunk 2]
```diff
@@ -9,16 +9,12 @@ public class LoginSession : Session
         protected override SessionType Type => SessionType.Login;
 
         public long AccountId;
+        public long CharacterId;
```
Comment (AlanMorel) [inline MapleServer2/Servers/Login/LoginSession.cs:12]: Where is this variable being used?

File: MapleWebServer/Program.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        public static void Main(string[] args)
+        {
+            WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+
+            builder.Services.AddControllers();
+            builder.Services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new() { Title = "MapleWebServer", Version = "v1" });
```
Comment (AlanMorel) [inline MapleWebServer/Program.cs:14]: Pull out the Swagger title and version to be used here and line 24

**Review by AlanMorel — CHANGES_REQUESTED** [review]

File: MapleServer2/Servers/Login/LoginSession.cs (hunk 2 above)
Comment (AngeloTadeucci) [reply MapleServer2/Servers/Login/LoginSession.cs:12]: In `LoginHandlerSession.cs`

File: MapleServer2/MapleServer.cs [hunk 4]
```diff
@@ -29,13 +30,14 @@ public static void Main()
             CultureInfo.CurrentCulture = new CultureInfo("en-US");
 
             // Load .env file
-            string dotenv = Path.Combine(Paths.SOLUTION_DIR, ".env");
+            string filePath = Path.Combine(Paths.SOLUTION_DIR, ".env");
```
Comment (AlanMorel) [inline MapleServer2/MapleServer.cs:33]: Isn't `filePath` less descriptive than `dotenv`?

**Review by AlanMorel — CHANGES_REQUESTED** [review]

File: MapleServer2/MapleServer.cs (hunk 4 above)
Comment (AngeloTadeucci) [reply MapleServer2/MapleServer.cs:33]: yeah you're right

**Review by AlanMorel — APPROVED** [review]

**Review by Sparkymod — APPROVED** [review]

**Comment by AngeloTadeucci** [comment]
#386

**Pull request merged**